relayer-cli/src/utils/relayerHelpers.ts

```diff
@@ -54,10 +34,7 @@ async function updateStateFile(chainId: number, createdTimestamp: number, nonceF
   };
   fs.writeFileSync(chain_state_file, JSON.stringify(json), { encoding: "utf8" });
 
-  const lockFileName = "./state/" + network + "_" + chainId + ".pid";
-  if (fs.existsSync(lockFileName)) {
-    fs.unlinkSync(lockFileName);
-  }
+  releaseLock(network, chainId);
```

Comment:
I believe we can remove `releaseLock` from here, and use it in `setupExitHandlers` during `cleanup`, [here](https://github.com/kleros/vea/blob/71c528085c53941ea625c995cee39774ce800f9b/relayer-cli/src/utils/relayerHelpers.ts#L43).

Reply:
@coderabbitai can you create an issue for this?